docker-integrations/tendermint/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build Stage 1
+# This build created a staging docker image
+#
+FROM node:lts AS appbuild
+WORKDIR /usr/src/app
+COPY ./package.json ./
+COPY ./tsconfig.json ./
+RUN yarn install
+COPY ./src ./src
+RUN yarn run build:binaries
+
+# Build Stage 2
+# This build takes the production build from staging build
+#
+FROM node:slim AS runtime
```

Comment:
@luis-rf Same here, let's use a specific tag or digest for the node image. Also it might be better to use a linux flavour since node is not needed in the runtime image given that we only start a binary.